docs/servicenow.md

```diff
@@ -162,12 +162,3 @@ one email address.
 ```python
 t.remove_cc(['username@domain.com', 'user3@domain.com'])
 ```
-
-### devops_one_url() <a name="devops_one"></a>
```

Comment:
Done and done, I have renamed and left STATE in the unit tests, but it's using new mapping.